### PR TITLE
Update to 0.11.0

### PR DESCRIPTION
**Pull request by frantsao**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change

- Updated to 0.11.0 version 
- Minor updates for ansible >= 2.7 and molecule >= 2.20.0
- Removed service pid file creation

### Benefits

We can run the latest mysqld_exporter version

### Possible Drawbacks
Can't install previous default version (0.10.0).

### Applicable Issues
#1

### CODE REVIEW

**Review by miguel-chacon — COMMENTED** [review]
Repository renaming breaks travis integration. Set it up again.

**Review by sorobon — APPROVED** [review]

**Review by jnogol — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-ansible==2.4.3.0
-molecule==2.10.1
-docker-py==1.10.6
+ansible==2.7.9.0
```
Comment (dortegau) [inline test-requirements.txt:1]: Readme file still refers to Ansible 2.4.3.0 :)

**Comment by frantsao** [comment]
> Repository renaming breaks travis integration. Set it up again.

Fixed itself :D

File: test-requirements.txt (hunk 1 above)
Comment (frantsao) [reply test-requirements.txt:1]: Fixed!

**Review by dortegau — APPROVED** [review]

**Pull request merged**